custom_components/audiconnect/audi_services.py

```diff
@@ -337,10 +337,12 @@ async def get_tripdata(self, vin: str, kind: str):
             key=lambda k: k["overallMileage"],
             reverse=True,
         )
-
+        # _LOGGER.debug("get_tripdata: td_sorted: %s", td_sorted)
         td_current = td_sorted[0]
+        _LOGGER.debug("TRIP DATA: td_current: %s", td_current)
         # FIX, TR/2023-03-25: Assign just in case td_sorted contains only one item
-        td_reset_trip = td_sorted[0]
+        td_reset_trip = td_sorted[1]
```

Comment:
@Kolbi Can you try changing this line back to:
```python
td_reset_trip = td_sorted[0]
```
This might resolve the long term trip if there is no reset trip